components/docs-chef-io/content/automate/notifications.md

```diff
@@ -13,23 +13,65 @@ gh_repo = "automate"
     weight = 10
 +++
 
-## About Notifications
+## Notifications
 
-Chef Automate notifications uses webhooks to send alerts for failures of Chef Infra Client runs and InSpec compliance scans. You can send notifications to Slack channels, Slack users, or any other service that supports webhook integrations. Notifications are sent for all failures. To ignore a failure, set the `ignore_failure` attribute to `true` on the resource. 
+Chef Automate notifications uses webhooks to send alerts for failures of Chef Infra Client runs and InSpec compliance scans. You can send notifications to Slack channels, Slack users, or any other service that supports webhook integrations. Notifications are sent for all failures. To ignore a failure, set the `ignore_failure` attribute to `true` on the resource.
 
 By default only Admins of Chef Automate may create and manage notifications.
 
 {{< warning >}}
 Notifications sent by Chef Automate do not support retries; therefore, notifications sent while Slack or the external service receiving the notification is experiencing API issues, outages, or some other unplanned downtime may never be received. Undelivered notifications are not re-sent. Attempts to send notifications do generate log messages in your Chef Automate server.
 {{< /warning >}}
 
+Chef Automate supports three types of notifications:
+
+- ServiceNow
+- Slack
+- Webhooks
+
+The steps to add different notifications to the Chef Automate is shown below:
+
+## ServiceNow Notification
+
+### Add a ServiceNow Notification
+
+To add a servicenow notification for Chef Automate, follow the steps given below:
+
+![ServiceNow notifications](/images/automate/servicenow-notifications-navigation.png)
+
+1. In the **Settings** tab, navigate to the _Notifications_ page in the sidebar.
+1. Select **Create Notification**.
+1. Select **ServiceNow**.
+1. Enter a unique notification name.
+1. Select the failure type to be notified on from the drop-down menu. Current options are Chef Infra Client run or InSpec scan.
+1. Get your ServiceNow webhook address by using the **What's this?** link, which opens an external servicenow site.
```

Comment:
Please check if this is available or not.

Reply:
Checked and updated accordingly.